docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1356,6 +1356,103 @@ See [transaction isolation levels](../../../architecture/transactions/isolation-
 
 Default: `read committed`
 
+## Catalog flags
```

Comment:
Why put catalog flags and advanced flags after PG server options? I would keep flags together and put options last
https://deploy-preview-21369--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-tserver/#postgresql-server-options